client/src/components/TextArea/TextArea.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import "./TextArea.css";
+
+const TextArea = props => (
+  <div>
+    <label className="text-area-label">{props.label}</label>
```

Comment:
Someone may or may not want to add a label - can you add a conditional to render this if the label is present?